vmware-base/src/main/java/com/cloud/hypervisor/vmware/util/VmwareHelper.java

```diff
@@ -549,12 +549,6 @@ public static void setBasicVmConfig(VirtualMachineConfigSpec vmConfig, int cpuCo
 
         cpuInfo.setReservation((long)cpuReservedMhz);
         vmConfig.setCpuAllocation(cpuInfo);
-        if (cpuSpeedMHz != cpuReservedMhz) {
```

Comment:
@DK101010 I think removing these lines will cause a regression on cpu and memory overcommit, have you tested those cases?

Reply:
Hi @nvazquez not sure what do you mean with regression.
Reservation and hot add are two different things. I can use reservation in it doesn't matter if hot add enabled/disbaled and it makes for me not sense to enable this when I'm reserve memory .
Also you can't more reserve as you have for the current vm.

I may be missing something, but currently I don't see a problem here.

Reply:
@DK101010 I have examined the `setBasicVmConfig(..)` method is used on:
- `VmwareResource.execute(StartCommand cmd)` -> the values set on these removed lines are not relevant, are overridden later (https://github.com/apache/cloudstack/blob/main/plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java#L1995-L2005)
- `HypervisorHostHelper.createBlankVm(..)` -> possible regression at this point: blank/worker VMs are always created with `cpuHotAdd = false` and `memHotAdd = false`. I'm not sure how/if this would affect deployments, any thoughts? 

cc. @sureshanaparti @rhtyd @DaanHoogland

Reply:
@nvazquez Hot-adding cpu & memory are set / overridden based on guest OS support. Also, the reservation and hot add are different things. So, I think, the hot add config here seems not relevant now.

Reply:
Thanks @sureshanaparti